draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,10 +627,39 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client MAY wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS ClientHello may be altered by the "esnikeys" value
+of an SVCB response {{svcparamkeys-esnikeys}}. CD (Connection Delay)
+is a configurable parameter. The recommended value is 50 milliseconds,
+as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+An SVCB record is consistent with an active or in-progress connection C
+if the following conditions are met:
+
+1. The A or AAAA address for its SvcDomainName matches the destination
+address of C.
+2. The destination port of C matches that specified in the SVCB record.
+
+If an SVCB record is consistent with an active or in-progress
+connection C, the client may continue using C with any information provided
+by the SVCB record. For example, if an SVCB record with a "esnikeys" value and
+"ipv4hint" or "ipv6hint" values {{svcparamkeys-iphints}} arrives, and one of
+those hints matches the address of an in-progress connection TCP connection,
+TLS should proceed on that connection using the provided ESNI keying material.
+
+If the SVCB record is not consistent with any active or in-progress connection,
+clients must then resolve the SvcDomainName in the SVCB response to the preferred
```

Comment:
Could you help me understand what the new requirement is here?  Or could we simplify this to "If the SVCB record is not consistent with ... clients must proceed as described in step 3 of `{{client-behavior}}`.

Reply:
Simplifying like so is perfectly fine by me! I wanted to keep the blurb to close the dangling "else" case in this section. Good suggestion. :)